shelley/chain-and-ledger/executable-spec/test/Generator/Utxo.hs

```diff
@@ -0,0 +1,143 @@
+{-# LANGUAGE BangPatterns #-}
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE PatternSynonyms #-}
+{-# LANGUAGE TypeSynonymInstances #-}
+{-# LANGUAGE UndecidableInstances #-}
+
+-- The HasTrace instance relies on test generators and
```

Comment:
Uh? Do we have an instance in this module?

Reply:
oops